### ISSUE TITLE
Cursor occasionally missing when opening a conversation

### ISSUE THREAD

**Issue by 2-4601** [WhisperSystems/Signal-Android#3144]
This is something I encountered while trying to reproduce #3134 of which I cut 'n' pasted my comments here because the issue appears to be separate.

Sometimes when I open a conversation the cursor is missing. This happens on average 5% of the time. The cursor comes back when you tap the input box though. I speculate this issue was introduced in 2.14(.1) but I'm not sure if I would have noticed anything without #3134.

Here are some tests I ran while trying to pin it down:
- for each conversation in the main conversation list
  * repeat (30, 50 or 100 times)
    - open the conversation
    - count if cursor shows or not
    - tap Android's back button to go back to main conversation list

**Nexus 4, Android 5.1, TS 2.14.1**
```
1st priv. conv. 1: 0/50 (ok after 50 tries nothing, i give up)
2nd priv. conv. 2: 3/30
3rd priv. conv. 3: 5/30
4th priv. conv. 4: 2/30
1st group conv. 1: 1/30
5th priv. conv. 5: 2/30
6th priv. conv. 6: 0/50
```
Result: in 13/250 cases I got a missing cursor. The conversations that got 0% do not really differ from the others.

**Huawei U8880, Android 4.0.4, TS 2.14.1**
```
1st priv. conv. 1: 0/100
```
Does not seem to happen on 4.0.4. But I had just one conversation.

Here's [a debug log](https://gist.github.com/002810e2f0f66373fe91) from Lollipop after:
- rebooting phone
- opening TS
- switching between the conversation list and a specific conversation until the cursor disappeared (took 16 tries)

I realize this is a _"new version of TextSecure makes my phone run 0.4 Fahrenheit colder than before and it's affecting my testicle balance when I keep my phone in my pocket"_ type of an issue but I'm doing this mostly for documentation. Maybe this has some relevance to another more important case in the future. I think I'm gonna unsub myself as soon as I hit that Submit button.

**Comment by Joesen1** [comment]
@2-4601 I've come over the same problem with TextSecure 2.14.2.
The cursor was missing the whole time!
Updated to newest 2.14.4 -> issue fixed. Good job, devs! ;-)

**Comment by 2-4601** [comment]
> The cursor was missing the whole time!

@Joesen1 In this issue the cursor is not missing the whole time. It comes back after tapping the input box. I think you might have mistaken this with #3134 which is already fixed in 2.14.4.

However this issue is still present in 2.14.4.

**Comment by 2-4601** [comment]
Closing because cannot reproduce any more on 3.3.1 (and because I hate this bug report).
